Jellyfin.Drawing.Skia/Jellyfin.Drawing.Skia.csproj

```diff
@@ -22,4 +22,16 @@
     <ProjectReference Include="..\MediaBrowser.Common\MediaBrowser.Common.csproj" />
   </ItemGroup>
 
+  <!-- Code analysers-->
+  <ItemGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.9.7" PrivateAssets="All" />
+    <PackageReference Include="SerilogAnalyzer" Version="0.15.0" PrivateAssets="All" />
+    <PackageReference Include="StyleCop.Analyzers" Version="1.1.118" PrivateAssets="All" />
+    <PackageReference Include="SmartAnalyzers.MultithreadingAnalyzer" Version="1.1.31" PrivateAssets="All" />
+  </ItemGroup>
```

Comment:
@Bond-009 I copied this code block from one of the other `.csproj` files. I know that analyzer packages are added by NuGet with the following configuration, since they are [development dependencies](https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#controlling-dependency-assets):

```
<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
<PrivateAssets>all</PrivateAssets>
```

Any reason we are not doing the same here?

Reply:
`PrivateAssets` is set as you can see and I don't see a reason to set `IncludeAssets`.

Reply:
Ah okay got it. I didn't notice that `PrivateAssests` is defined inline as an attribute.